File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/KingOfTheHill/AnswerResultData.cs
namespace GeoQuiz_backend.Application.DTOs.KingOfTheHill
{
    public class AnswerResultData
    {
        public bool IsCorrect { get; set; }
        public int ScoreGained { get; set; }
        public int TimeSpentMs { get; set; }
        public int RemainingPlayers { get; set; }
        public int CorrectOptionIndex { get; set; } 
    }
}

File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/koth/AnswerResultData.java
package com.example.geoquiz_frontend.data.remote.dtos.koth;

public class AnswerResultData {
    private boolean isCorrect;
    private int scoreGained;
    private int timeSpentMs;
    private int remainingPlayers;
    private int correctOptionIndex;

    public boolean isCorrect() { return isCorrect; }
    public int getScoreGained() { return scoreGained; }
    public int getTimeSpentMs() { return timeSpentMs; }
    public int getRemainingPlayers() { return remainingPlayers; }
    public int getCorrectOptionIndex() { return correctOptionIndex; }
}
